### ISSUE TITLE
Unable to generate Inline Response Schemas

### ISSUE THREAD

**Issue by kkwanpsn** [Mermade/widdershins#175]
With widdershins ver. 3.6.2, if I call `node widdershins.js` with the `--resolve` option on a OAS 3.0 YAML file containing externally referenced schemas, Response Schema markdown is no longer resolved and/or generated inline.

Using the following OAS 3.0 YAML file [petstore-openapi-extref.yaml](https://gist.github.com/kkwanpsn/c95c47fc3cc21708b33eeedc1e78b8ec#file-petstore-openapi-extref-yaml) with external references and [petstore-openapi-defs.yaml](https://gist.github.com/kkwanpsn/c95c47fc3cc21708b33eeedc1e78b8ec#file-petstore-openapi-defs-yaml) containing the referenced schemas, I execute the following:

`$ node widdershins.js --resolve petstore-openapi-extref.yaml -o petstore.md`

The output generated is missing response schema markdown. See [petstore-openapi-markdown.diff](https://gist.github.com/kkwanpsn/c95c47fc3cc21708b33eeedc1e78b8ec#file-petstore-openapi-markdown-diff) diff file, comparing it to the markdown generated by widdershins ver. 3.5.0.

When rendering the widdershins ver. 3.6.2 markdown, in slate it's easier to see only expanded paths strings under the "Schema" column of the "Responses" section:

![screen shot 2018-09-24 at 1 47 28 pm](https://user-images.githubusercontent.com/26638305/45979902-62d07c00-c005-11e8-8ea7-dca4b78b4c71.png)

Where as markdown generated by widdershins ver. 3.5.0 produces the "Inline" string value, referencing the expanded schema also included in the "Response Schema" section below it:

![screen shot 2018-09-24 at 1 48 33 pm](https://user-images.githubusercontent.com/26638305/45980067-e1c5b480-c005-11e8-9719-f93d55ffa7f3.png)

Notes:
Related to Issue No. [172](https://github.com/Mermade/widdershins/issues/172)

**Comment by MikeRalphson** [comment]
Thanks @kkwanpsn for the detailed report.

This was due to a change in the way OAS documents are resolved (earlier versions of `swagger2openapi` duplicated the contents of `$ref`documents, whereas `oas-kit`'s `oas-resolver` creates internal `$ref`s to the first place content is included).

As a note, if you **don't** want this behaviour, the simplest way to structure your OAS documents is to use `$ref`s to `#/components/schemas/schemaName` and have the definition of `schemaName` do the actual `$ref` to the external definition:

```yaml
components:
  schemas:
    schemaName:
      $ref: './my-openapi-definitions.yaml#/components/schemas/schemaName'
```

Should be fixed in `v3.6.3`.

**Comment by kkwanpsn** [comment]
Thank you @MikeRalphson for your prompt response. Appreciate it!

**Comment by timothymcmackin** [comment]
@MikeRalphson By "Should be fixed in v3.6.3." do you mean that I should no longer have to "$refs to #/components/schemas/schemaName and have the definition of schemaName do the actual $ref to the external definition"?
